test/spec/modules/rubiconBidAdapter_spec.js

```diff
@@ -271,29 +257,29 @@ describe('the rubicon adapter', () => {
           expect(query['alt_size_ids']).to.equal('57,59');
         });
 
-        it('should not send a request and register an error bid if no valid sizes', () => {
+        it('should not send a request if no valid sizes', () => {
           var sizesBidderRequest = clone(bidderRequest);
           sizesBidderRequest.bids[0].sizes = [[620, 250], [300, 251]];
 
+          sandbox.stub(spec, 'buildRequests');
+
           rubiconAdapter.callBids(sizesBidderRequest);
```

Comment:
This test can be easily tested by using `spec.isBidRequestValid` and `spec.buildRequests`. Call to `callBids` can be removed.

Same goes for other tests in `for request` test suite. They can be tested using spec functions.